`col` is no longer available in `util-linux`, and these
txt files aren't installed into the final package anyway

--- a/Makefile
+++ b/Makefile
@@ -70,7 +70,7 @@
 fmt:
 	go fmt ./...
 
-man-pages: $(HELP_ALL:=.md) $(HELP_ALL) $(HELP_ALL:=.txt)
+man-pages: $(HELP_ALL:=.md) $(HELP_ALL)
 
 %.txt: %
 	groff -Wall -mtty-char -mandoc -Tutf8 -rLL=$(TEXT_WIDTH)n $< | col -b >$@
